# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/metamail-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mm2.7/src/richmail/richlex.c.orig	2005-01-25 12:45:17.000000000 -0800
+++ mm2.7/src/richmail/richlex.c	2005-01-25 12:45:19.000000000 -0800
@@ -49,10 +49,12 @@
 int CorrectionEnabled = 1;	/* Zero if correction has been disabled */
 int RichtextLessThanFlag = 0;	/* Non-zero to turn on multi-byte '<' hack */
 
+#if 0
 #ifndef AMIGA
 extern	int fgetc ();
 extern	int fputc ();
 #endif
+#endif
 
 int (*RichtextGetc) () = fgetc;	/* Function to call to get characters */
 int (*RichtextPutc) () = fputc; /* Function to call to put characters */
